extension/package.json

```diff
@@ -10,9 +10,10 @@
         "Uchida Taishi"
     ],
     "version": "0.7",
-    "fullName": "MemChaser",
+    "title": "MemChaser",
     "id": "memchaser@quality.mozilla.org",
     "description": "Chasing garbage collection and memory usage",
+    "main": "lib/main.js",
```

Comment:
There is a nice [migration page][1] on MDN. I based my work on that.


[1]: https://developer.mozilla.org/en-US/Add-ons/SDK/Tools/cfx_to_jpm